jenkins_jobs/periodic_pipelines.groovy

```diff
@@ -0,0 +1,23 @@
+def createPeriodicPipeline(String name) {
+  pipelineJob(name) {
+    definition {
+      cps {
+        script("""\
+node('master') {
+  dir('infrastructure') {
+    git(url: 'ssh://git@github.com/$GITHUB_ORGANIZATION_NAME/infrastructure.git',
+        branch: 'pipeline')
+  }
+  pipeline = load 'infrastructure/pipeline/${name}.groovy'
+}
+
+pipeline.execute()
+""")
+        sandbox()
```

Comment:
It allows the execution of the script, as long as the methods used are approved.
Without this option, an admin would have to manually approve the whole script each time it changes.
https://support.cloudbees.com/hc/en-us/articles/207406207-Avoid-script-approvals-with-a-Jenkins-Pipeline-Groovy-script